### PR TITLE
Adding email links

### PR DESCRIPTION
**Pull request by gbinal**
In support of #844.

### CODE REVIEW

File: pages/index.html [hunk 1]
```diff
@@ -187,6 +187,6 @@ <h3 class="usa-example-heading usa-link-external">
     <div class="usa-cta">
       <a class="usa-button usa-button-secondary" href="{{ site.repos[0].url }}">Contribute on GitHub</a>
     </div>
-    <a href="{{ site.baseurl }}/getting-started" onclick="ga('send', 'event', 'Viewed the standards', 'Clicked View the Standards link at the bottom of the homepage');">Or view the standards</a>
+    <a href="mailto:uswebdesignstandards@gsa.gov">Or drop us an email</a>
```
Comment (maya) [inline pages/index.html]: Do we want to include any GA handler events in this? cc @colinpmacarthur

**Comment by carodew** [comment]
Can we write out the full email address? Many people don't have their primary email client set up on the computer and clicking on the email link opens an email client they don't use and haven't set up an account with. Writing out the email address means they can also copy/paste into their preferred email  client.

**Comment by gbinal** [comment]
We definitely could.  It's a bit lengthy though and may not look good.  I don't see it as a big problem but am fine either way.

**Comment by maya** [comment]
I've updated the footer to state the full email address.

What do you guys prefer for the homepage on the bottom?

screen shot 2015-10-21 at 11 54 51 am
screen shot 2015-10-21 at 11 54 34 am

**Comment by mollieru** [comment]
@maya the second option is compliant w our link styles (https://playbook.cio.gov/designstandards/visual-style/#links) so lets go w that one!

**Comment by maya** [comment]
@mollieru done in b078614. Also, adding the extra space resolves #552.

**Pull request merged**